examples/mamba/mamba.py

```diff
@@ -813,6 +805,14 @@ def forward(
             label_mask=label_mask,
         )["loss"]
         return {"loss": loss}
+    
+    def get_named_params_without_weight_decay(self):
```

Comment:
why is it inside masked_mean?

Reply:
?

https://github.com/huggingface/nanotron/blob/2f9e7d8884202c7b72294aacc1d55dfe1cd43100/examples/mamba/mamba.py#L809-L815

Reply:
it's in the NanotronModel

Reply:
sorry was reviewing through github (displayed was buggy). Anyway, we could have kept it in `MambaModel` no ?